src/components/Select/components/SelectPopup/SelectPopup.scss

```diff
@@ -78,6 +78,7 @@ $xl-hor-padding: '12px';
         display: flex;
         align-items: center;
         width: 100%;
+        height: 100%;
```

Comment:
It needs to avoid problems such this
<img width="736" alt="Снимок экрана 2022-03-28 в 18 59 20" src="https://user-images.githubusercontent.com/26501073/164085116-9123277e-f90b-4bdf-b079-3299ac451167.png">